e2e/cypress/tests/integration/account_settings/profile/email_spec.ts

```diff
@@ -156,7 +172,7 @@ describe('Profile > Profile Settings > Email', () => {
             // # Wait for one second for the mail to be sent out.
             cy.wait(TIMEOUTS.FIVE_SEC);
 
-            cy.getRecentEmail(testUser).then(({subject}) => {
+            cy.getRecentEmail({username, email}).then(({subject}) => {
```

Comment:
This should be reverted using `testUser` where it has the old email address.

Reply:
my bad, actually I completely misread the tests